### PR TITLE
DROOLS-4945 : Deficient Rows warning shows when Decision Table references items in a list

### PR DESCRIPTION
**Pull request by Rikkola**
https://issues.redhat.com/browse/DROOLS-4945

Bundle:
https://github.com/kiegroup/drools/pull/2727
https://github.com/kiegroup/drools-wb/pull/1283

### CODE REVIEW

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Comment by Rikkola** [comment]
@jomarko This PR should now build the webworker js.

**Comment by Rikkola** [comment]
But this needs to be retriggered 
jenkins execute full downstream build

File: drools-wb-services/drools-wb-verifier/drools-wb-verifier-webworker/pom.xml [hunk 1]
```diff
@@ -77,6 +77,7 @@
         <groupId>org.codehaus.mojo</groupId>
         <artifactId>gwt-maven-plugin</artifactId>
         <configuration>
+          <skip>false</skip>
```
Comment (Rikkola) [inline drools-wb-services/drools-wb-verifier/drools-wb-verifier-webworker/pom.xml]: This does do the trick, but not sure if this is the best way. For some reason drools-wb-showcase runs by GWT by default, but there is no easy ( for me ) to spot reason why the webworker should not do the same. Spending some more time on investigating this.

**Comment by Rikkola** [comment]
@manstis if you could please review.

**Comment by Rikkola** [comment]
jenkins execute full downstream build

**Pull request merged**